chain/provider.go

```diff
@@ -26,6 +26,10 @@ type Chain interface {
 	TransferICST(ctx context.Context, from []byte, to []byte, value *big.Int) (*transaction.Transaction, error)
 	WatchBlocks(ctx context.Context, blockStart *big.Int) (<-chan *transaction.Block, <-chan error)
 	WatchICSTTransfer(ctx context.Context, blockStart *big.Int) (<-chan *transaction.Block, <-chan error)
+	Plasma
+}
+type Plasma interface {
```

Comment:
not a good idea to change `go-icst`. better to use a new interface to wrap `Chain` in project `Plasma`